hw/top_earlgrey/doc/_index.md

```diff
@@ -16,24 +16,33 @@ This is not a specification of the final implementation.
   - DM (debug module)
```

Comment:
is it worth mentioning lock-step cores here? what do you think?

Reply:
I would mention it, since it is a pretty salient security feature of the processor.

Reply:
Done. Added a short list of Ibex security features.